src/applications/personalization/view-dependents/components/ViewDependentsHeader/ViewDependentsHeader.jsx

```diff
@@ -1,22 +1,34 @@
-import React from 'react';
+import recordEvent from 'platform/monitoring/record-event';
+import React, { Component } from 'react';
 
-const ViewDependentsHeader = () => (
-  <div className="vads-l-row">
-    <div className="vads-l-col--12">
-      <h1>Your VA Dependents</h1>
-      <p className="vads-u-font-size--md vads-u-font-family--serif">
-        Below is a list of dependents we have on file for you. You can file a
-        claim for additional disability compensation whenever you add a new
-        dependent.
-      </p>
-      <a
-        href="https://www.ebenefits.va.gov/ebenefits/about/feature?feature=dependent-compensation"
-        className="usa-button-primary va-button-primary"
-      >
-        Add or remove a dependent
-      </a>
-    </div>
-  </div>
-);
+class ViewDependentsHeader extends Component {
+  handleClick = () => {
+    recordEvent({
+      event: 'cta-default-button-click',
+    });
```

Comment:
Not necessary at all right now but if we find ourselves making a bunch of CTA buttons to eBenefits we may want to make a super simple component to abstract the GA event and maybe enforce the URL structure (`?feature=...`)

Reply:
I will bring this up with the Analytics team